doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
2015-05-23 10:16 GMT+02:00 Raffi Enficiaud <notifications@github.com>:

> In doc/test_organization/managing_tests_dependencies.qbk
> <https://github.com/boostorg/test/pull/64#discussion_r30943082>:
>
> > @@ -7,17 +7,31 @@
> >
> >  [section:tests_dependencies Managing test dependencies]
> >
> > -Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators].
> > +In general, you should make sure that test cases are independent on one another.
>
> I would rather put the emphasis on the order: users should not rely on any
> execution order, and tests execution can be shuffled.
>

Are you suggesting to only change the wording from imperative "you should
not make the tests dependent" to a more informative "you should not rely on
the ordering"?

If so, I do not see any problem with such change.

Regards,
&rzej